.travis.yml

```diff
@@ -138,49 +138,45 @@ matrix:
         - ./bootstrap.sh -e -t && cd build  && cmake -DUSE_SHARED_LIBS=ON .. &&  cmake --build . --parallel 4  && sudo make test ARGS="-j4 --output-on-failure"
     - os: osx
       osx_image: xcode10.3
+      addons:
```

Comment:
According to the [documentation](https://docs.travis-ci.com/user/installing-dependencies/#installing-packages-on-macos), this won't update the packages, only if specified explicitly. So far we did update them. If the packages in the Travis VM snapshot are recent enough this should be ok, and we can make builds faster this way, but we should make sure of that.

Reply:
Yep. As the images were updated recently, I think we should be fine with the snapshots.
Will see the CI results.